csrc/register_interface.cpp

```diff
@@ -607,7 +610,7 @@ torch::jit::RegisterOperators reg_infer_index_select({
             auto idx_size = torch::jit::pop(stack).toIntVector();
             auto size = torch::jit::pop(stack).toIntVector();
             size[selected_dim] = idx_size[0];
-            torch::jit::push(stack, c10::IValue(std::move(size)));
+            torch::jit::push(stack, c10::IValue(size));
```

Comment:
Why `std::move` is removed?

Reply:
Error (CLANGTIDY) [performance-move-const-arg,-warnings-as-errors] passing result of std::move() as a const reference argument; no move will actually happen

        607  |            auto idx_size = torch::jit::pop(stack).toIntVector();
        608  |            auto size = torch::jit::pop(stack).toIntVector();
        609  |            size[selected_dim] = idx_size[0];
    >>> 610  |            torch::jit::push(stack, c10::IValue(std::move(size)));
        611  |          };
        612  |        },
        613  |        torch::jit::aliasAnalysisFromSchema()),